image/templates/helm/stackrox-central/config/proxy-config.yaml.default

```diff
@@ -9,6 +9,7 @@
 # # - *.stackrox, *.stackrox.svc
```

Comment:
> wil NOT be excluded

Wait isn't it the opposite? I.e. by default these hosts will be added to `NO_PROXY`?

Reply:
Indeed, when `omitDefaultExcludes: false` then default excludes will not be ommited and added to no proxy while when it's true it wont'